usaspending_api/etl/tests/integration/test_load_multiple_submissions.py

```diff
@@ -248,25 +248,27 @@ def setUp(self):
                     object_class,
                     gross_outlay_amount_by_awa_cpe,
                     transaction_obligated_amou,
+                    ussgl487200_downward_adjus_cpe,
+                    ussgl497200_downward_adjus_cpe,
                     disaster_emergency_fund_code
                 ) (values
-                    (1, 1, 1, '1101', 11111, 111110, null),
-                    (2, 1, 1, '1101', 22222, 222220, 'B'),
-                    (3, 1, 1, '1101', 33333, 333330, 'L'),
-                    (4, 2, 1, '1101', 44444, 444440, null),
-                    (5, 2, 1, '1101', 55555, 555550, null),
-                    (6, 2, 1, '1101', 66666, 666660, null),
-                    (7, 3, 2, '1101', 77777, 777770, 'L'),
-                    (8, 3, 2, '1101', 88888, 888880, 'L'),
-                    (9, 3, 2, '1101', 99999, 999990, 'L'),
-                    (10, 4, 2, '1101', 10101, 101010, null),
-                    (11, 5, 2, '1101', 11111, 111110, 'B'),
-                    (12, 5, 2, '1101', null, null, 'B'), -- this should not load because of 0/null values
-                    (13, 5, 2, '1101', 0, 0, 'B'), -- this should not load because of 0/null values
-                    (14, 5, 2, '1101', null, 0, 'B'), -- this should not load because of 0/null values
-                    (15, 5, 2, '1101', 0, null, 'B'), -- this should not load because of 0/null values
-                    (16, 6, 2, '1101', 12121, 121210, 'L'),
-                    (17, 7, 2, '1101', 13131, 131310, 'N')
+                    (1, 1, 1, '1101', 11111, 111110, -11, -111, null),
+                    (2, 1, 1, '1101', 22222, 222220, -22, -222, 'B'),
+                    (3, 1, 1, '1101', 33333, 333330, -33, -333, 'L'),
+                    (4, 2, 1, '1101', 44444, 444440, -44, -444, null),
+                    (5, 2, 1, '1101', 55555, 555550, -55, -555, null),
+                    (6, 2, 1, '1101', 66666, 666660, -66, -666, null),
+                    (7, 3, 2, '1101', 77777, 777770, -77, -777, 'L'),
+                    (8, 3, 2, '1101', 88888, 888880, -88, -888, 'L'),
+                    (9, 3, 2, '1101', 99999, 999990, -99, -999, 'L'),
+                    (10, 4, 2, '1101', 10101, 101010, -10, -101, null),
+                    (11, 5, 2, '1101', 11111, 111110, -11, -111, 'B'),
+                    (12, 5, 2, '1101', null, null, 0, 0, 'M'), -- this should not load because of 0/null values
+                    (13, 5, 2, '1101', 0, 0, null, 0, 'M'), -- this should not load because of 0/null values
+                    (14, 5, 2, '1101', null, 0, null, 0, 'M'), -- this should not load because of 0/null values
+                    (15, 5, 2, '1101', 0, null, 0, null, 'M'), -- this should not load because of 0/null values
```

Comment:
The only thing I can think of adding to this PR is some rows here with a combination of null/0 values with numeric values to test that rows with at least one numeric value gets included in the aggregation. Not a blocker for approval.

Reply:
Updated tests

Reply:
Looks good